modules/parachains/src/lib.rs

```diff
@@ -23,47 +23,87 @@
 
 #![cfg_attr(not(feature = "std"), no_std)]
 
-use bp_polkadot_core::parachains::{ParaHash, ParaHasher, ParaHead, ParaId, ParachainHeadsProof};
```

Comment:
@jiguantong There are lots of changes in the parachain pallet, please help take a review.